QgisModelBaker/libqgsprojectgen/generator/generator.py

```diff
@@ -236,8 +236,22 @@ def relations(self, layers, filter_layer_list=[]):
                 layers)
             relations = relations + domain_relations
         else:
+            """Create the bags_of_enum structure"""
```

Comment:
```suggestion
            # Create the bags_of_enum structure
```

The other notation is mainly for docstrings